aus_council_scrapers/base.py

```diff
@@ -21,132 +21,14 @@
     TIME_REGEX,
     TIMEZONES_BY_STATE,
 )
+from .data import ScraperResult, NoticeDate, NoticeLocation, NoticeHydrationInputs
 
 
 def register_scraper(cls):
     SCRAPER_REGISTRY[cls.__name__] = cls()
     return cls
 
 
-@dataclass
-class ScraperReturn:
-    """Designates what a scraper should return.\n
-    If a given item in the scraper is None, it will be skipped.\n
-    `name`: The name of the meeting (e.g. City Development Delegated Committee).\n
-    `date`: The date of the meeting (e.g. 2021-08-01).\n
-    `time`: The time of the meeting (e.g. 18:00).\n
-    `webpage_url`: The URL of the webpage where the agenda is found.\n
-    `download_url`: The URL of the PDF of the agenda.\n
-    `location`: The location of the meeting (e.g. Council Chambers).\n
-    `cleaned_time`: The time of the meeting as a time object.\n
-    `cleaned_date`: The date of the meeting as a date object.\n
-    """
-
-    name: Optional[str]
-    date: str
-    time: Optional[str]
-    webpage_url: str
-    download_url: str
-    location: Optional[str] = None
-
-    # Cached properties
-    _cleaned_time: Optional[datetime.time] = None
-    _cleaned_date: Optional[datetime.date] = None
-
-    @property
-    def cleaned_time(self) -> Optional[datetime.time]:
-        try:
-            if not self.time:
-                return None
-            if not self._cleaned_time:
-                self._cleaned_time = parse_date(self.time, fuzzy=True).time()
-            return self._cleaned_time
-        except Exception as e:
-            return None
-
-    @property
-    def cleaned_date(self) -> datetime.date:
-        if not self.date:
-            raise ValueError("Date is required")
-
-        try:
-            if not self._cleaned_date:
-                self._cleaned_date = parse_date(self.date, fuzzy=True).date()
-        except Exception as e:
-            raise ValueError(f"Could not parse date {self.date}")
-
-        return self._cleaned_date
-
-    @property
-    def cleaned_location(self) -> Optional[str]:
```

Comment:
moved to `aus_council_scrapers.data.NoticeLocation.Raw`

I think that was unnecessary tbh. I just wanted to stuff more flexible for the scraper but also kind of encapsulate this logic